### PR TITLE
provider/google: Clone `regional` attribute as well.

### PR DESCRIPTION
**Pull request by duftler**

### CODE REVIEW

**Comment by duftler** [comment]
@lwander please review.

**Comment by duftler** [comment]
@lwander please review.

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy [hunk 1]
```diff
@@ -100,6 +100,10 @@ class CopyLastGoogleServerGroupAtomicOperation implements AtomicOperation<Deploy
 
     // Override any ancestor values that were specified directly on the cloneServerGroup call.
     newDescription.region = description.region ?: Utils.getLocalName(ancestorServerGroup.region)
+    newDescription.regional =
+        description.regional != null
+        ? description.regional
+        : ancestorServerGroup.regional
```
Comment (lwander) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/CopyLastGoogleServerGroupAtomicOperation.groovy]:
```
? ...
: ...
```

you madman

**Comment by lwander** [comment]
LGTM

**Pull request merged**